test/message-generator/test/translation-proxy.json

```diff
@@ -12,18 +12,27 @@
         "You can use any miner you want, by changing the execution command or by launching it separately"
     ],
     "common_messages": [
+        {
+            "message": {
+                "type": "SetupConnectionSuccess",
+                "flags": 300,
+                "used_version": 180
+            },
+            "replace_fields": [["flags", "goofy"],["used_version", "pluto"]],
+            "id": "setup_connection_success_tproxy"
+        }
     ],
     "mining_messages": [
         {
             "message": {
                 "type": "OpenExtendedMiningChannelSuccess",
-                "request_id": 666666,
+                "request_id": 666,
                 "channel_id": 1,
-                "target": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 255, 255, 255, 255, 255, 255, 255, 255],
+                "target": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 55, 55, 55, 55, 25, 25, 0, 0],
                 "extranonce_size": 16,
                 "extranonce_prefix": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1]
             },
-            "replace_fields": [["request_id","variable_name"]],
+            "replace_fields": [["request_id","variable_name"], ["target", "daisy"]],
```

Comment:
As var name it would be better to use something meaningful eg: open_extended_mining_channel_response_request_id or open_extd_m_channel_req_id

This is valid for all the variable_name in this PR

Reply:
ok now the labels should be ok!